roles/openshift_node/meta/main.yml

```diff
@@ -13,6 +13,4 @@ galaxy_info:
   - cloud
 dependencies:
 - role: lib_openshift
-- role: openshift_cloud_provider
-  when: not (openshift_node_upgrade_in_progress | default(False))
```

Comment:
Are you accounting for this somewhere else?

Reply:
@kwoodson Do you not see the other two files in this PR?

Reply:
@michaelgugino, I see them but they do not explain the logic that is missing. 

This is an entry point for configure nodes.
`playbooks/openshift-node/private/configure_nodes.yml` 

This is now also an entry point for containerized nodes:
`playbooks/openshift-node/private/containerized_nodes.yml`

The logic that was in this when statement was to protect nodes that are performing an upgrade.  The `openshift_cloud_provider` now lives in 2 more files.  If you've separated this content out and removed it from the upgrade path then please state such in the description or code comment.  If you are accounting for this in the hosts pattern (`- hosts: oo_hosts:!oo_hosts_to_upgrade` then please explicitly state such.  This will simplify the PR review process and get code merged much more efficiently.

Please don't assume that others know the intentions of code not mentioned in the current PR as the current PR doesn't state how it is dealing with the upgrade logic path of running the node role.

Happy holidays.

Reply:
@kwoodson I'm not sure what you mean.

Currently, openshift_node_upgrade_in_progress is set to "True" when this role is called during upgrades (or rather, parts of the role via include_role).

Now that we're mentioning it, it appears that boolean only exists for this condition now.  I'll go ahead and update this patch to remove the references to it.